qutebrowser/config/configdata.yml

```diff
@@ -975,6 +975,71 @@ scrolling.smooth:
 
     Note smooth scrolling does not work with the `:scroll-px` command.
 
+## spell
+
+spell:
+  type: Bool
+  default: true
+  desc: Enable spell checking.
+  backend:
+    QtWebKit: false
+    QtWebEngine: Qt 5.8
+
+spell_languages:
```

Comment:
Maybe that should be `spell.enabled` and `spell.languages`? I also wonder whether it'd be better to call it `spellcheck` instead...